python/sphinx_docs/pyproject.toml

```diff
@@ -11,6 +11,7 @@ repository = "https://github.com/OPM/opm-simulators"
 
 [tool.poetry.dependencies]
 python = "^3.10"
+requests = ">=2.32.4"
```

Comment:
Maybe we should use the caret operator instead of ">=" operator? To be consistent with the other package requirements which use the caret operator